win32ss/gdi/eng/surface.c

```diff
@@ -127,6 +127,10 @@ SURFACE_AllocSurface(
     PSURFACE psurf;
     SURFOBJ *pso;
     PVOID pvSection;
+    BOOL bExtraBitsNeeded = FALSE;
+
+    DPRINT("cy '%ld' iFormat '%ld' cjWidth '%ld' cjBufSize '%ld' pvBits '%p'.\n",
```

Comment:
Omit the trailing dot in the logging please.

Reply:
Done. Thanks.